bugbug/model.py

```diff
@@ -455,6 +456,19 @@ def train(self, importance_cutoff=0.15, limit=None):
         y_pred = self.clf.predict(X_test)
         y_pred = self.le.inverse_transform(y_pred)
 
+        # calibrating the model
+
+        # Fit isotonic regression model to the predicted probabilities
+        iso_reg = IsotonicRegression(out_of_bounds="clip")
+        iso_reg.fit(y_pred, y_test)
+
+        # Use the isotonic regression model to transform the predicted probabilities
+        calibrated_y_pred = iso_reg.transform(y_pred)
```

Comment:
The calibrated probabilities are still not used

Reply:
So everywhere i should replace the probabilities of the  XGBoost model by the calibrated probabilities?

Reply:
We need to output the calibrated probabilities to be used as a confidence level. 

See: https://github.com/mozilla/bugbug/blob/f9906057a5281b8913fb5a92edbe73440953581b/bugbug/model.py#L584

Reply:
@suhaibmujahid im not really sure how to do that!

Reply:
You need to use the trained isotonic model here. Our goal is to use its output as a confidence level.

Reply:
can you tell me what exactly is the function "classify" doing and how can I use the calibrated probabilities as confidence level in this function :sweat_smile: :

Reply:
ok so I tried something, just take a look!